apache-hbase-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,113 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'apache-hbase-alerts',
+        rules: [
+          {
+            alert: 'HighHeapMemUsage',
+            expr: |||
+              100 * sum without(context, hostname, processname) (jvm_metrics_mem_heap_used_m / clamp_min(jvm_metrics_mem_heap_committed_m, 1))  > %(alertsHighHeapMemUsage)s
```

Comment:
These metrics look generic too me. To make sure we don't leave chances that there could be other integration having same java metrics (instrumented same way as Hbase), best if we add filtering selector to each alert rule. That we we could be confident that there would no potential overlaps. And Hbase alerts woudn't fire on some other JVMs unrelated to Hbase.

Reply:
I understand what you mean, but couldn't find any examples of this in other mixins. Made an attempt in the most recent commit, let me know if it needs to be changed again.